source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -42,7 +42,7 @@ The following changes apply for Operator v5.0.0 or later:
 - The ``.spec.s3`` field is replaced by the ``.spec.features`` field.
 - The ``.spec.credsSecret`` field is replaced by the ``.spec.configuration`` field.
 
-  The ``.spec.credsSecret`` should hold all the environment variables for the MinIO deployment that contain sensitive information and should not show in ``.spec.env``.
+  The ``.spec.configuration`` secret should hold all the environment variables for the MinIO deployment that contain sensitive information and should not show in ``.spec.env``.
```

Comment:
maybe something like this?
```suggestion
  The ``.spec.configuration`` secret holds sensitive environment variables for the MinIO deployment [such as?]
To protect this information, these environment variables are not shown in ``.spec.env``.
```

Reply:
It is in fact *all* environment configuration settings. So you could link to the docs page on server envvars.

Reply:
_Technically_ the `.spec.env` can potentially work, but this is what we recommend.

Reply:
Reworked to take in these suggestions.

Reply:
Actually, the PR for 6.0.0 upgrades completely removed the need for this change.